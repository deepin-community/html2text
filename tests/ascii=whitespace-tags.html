<!doctype html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en-US">
<body>
	<foo/>
	<?xml:namespace prefix="o" ns="urn:schemas-microsoft-com:office:office" />
<b>a </b>test!
<b> a</b>test!
<b>a
</b>test!<br>
<br>
  test
  <div>
		<b>this</b>is<u>some</u>text
		<p />
		<b>this</b> is <u>the same</u> text
		<p />
		<a href="http://goo">that</a> <b>is</b> a link
		<p />
		<font face="Arial"><font color="#000080">This email</font>
			<a href="http://goo">is fake</a>
  </div>
</body>
</html>
