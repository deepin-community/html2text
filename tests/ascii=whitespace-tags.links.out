aa test! aatest! aa test!

test
tthhiissis_s_o_m_etext
tthhiiss is _t_h_e_ _s_a_m_e text
_t_h_a_t[1] iiss a link
This email _i_s_ _f_a_k_e[2]


############ RReeffeerreenncceess ############
   1. http://goo
   2. http://goo
